source/clear-linux/get-started/virtual-machine-install/virtualbox-cl-installer.rst

```diff
@@ -3,256 +3,306 @@
+   .. figure:: figures/vbox/virtualbox-cl-installer-03.png
+      :scale: 100%
+      :alt: Create a virtual hard disk
+
+      Figure 3: Create a virtual hard disk
+
+#. Select :guilabel:`Create`.
+
+#. In :guilabel:`Hard disk file type`, select
+   :guilabel:`VDI (VirtualBox Disk Image)`.
+
+   .. figure:: figures/vbox/virtualbox-cl-installer-04.png
+      :scale: 100%
+      :alt: Hard disk file type
+
+      Figure 4: Hard disk file type
+
+#. Select :guilabel:`Next`
+
+#. In :guilabel:`Storage on physical hard disk`, select
+   :guilabel:`Dynamically allocated`.
 
-   - File size: **8.00 GB** (this can be adjusted appropriately)
-   
-   - Hard disk file type: **VDI (Virtual Box Disk Image)**
+   .. figure:: figures/vbox/virtualbox-cl-installer-05.png
+      :scale: 100%
+      :alt: Storage on a physical hard disk
 
-   - Storage on physical hard disk: **Dynamically allocated** 
+      Figure 5: Storage on a physical hard disk
 
-   .. image:: ./figures/vbox/vbox-create-disk.png
-      :alt: Create a new virtual hard disk in VirtualBox
+#. Select :guilabel:`Next`.
+
+#. In :guilabel:`File location and size`, select a File size: **8.00 GB**
+   This can be adjusted appropriately.
+
+   .. figure:: figures/vbox/virtualbox-cl-installer-06.png
+      :scale: 100%
+      :alt: File location and size
+
+      Figure 6: File location and size
+
+   .. note::
+
+      If using the :ref:`bare-metal-install-desktop`, we recommend a starting point around 30 GB. The file size can be decreased in the future.
 
 #. Click the *Create* button.
 
-#. A new virtual machine will be created and appear in the |VBM|. Click 
-   *Settings* to configure the |CL| VM.
+   A new virtual machine will be created and appear in the |VBM|.
 
-   .. image:: ./figures/vbox/vbox-vm-created.png
-      :alt: A VM selected in VirtualBox Manager
+#. Click *Settings*, or :guilabel:`gear` icon, to configure the |CL| VM.
 
-#. A *VM - Settings* window will appear. Navigate to the *System* pane from 
-   the left-hand and select the following setting:
+#. In the left-hand menu, navigate to the :guilabel:`System` menu and check
+   the following settings:
 
    - **Enable I/O APIC**
    - **Enable EFI (special OSes only)**
-   
 
-   .. image:: ./figures/vbox/vbox-vm-settings-EFI.png
-      :alt: Enable EFI on a VirtualBox VM settings
+   .. figure:: figures/vbox/virtualbox-cl-installer-07.png
+      :scale: 100%
+      :alt: Settings>System
+
+      Figure 7: Settings>System
 
 .. note::
-   By default, only 1 virtual CPU is allocated to the new VM. Consider 
-   increasing the number of virtual processors allocated to the virtual 
-   machine under Settings --> System --> Processor for increased 
+
+   By default, only 1 virtual CPU is allocated to the new VM. Consider
+   increasing the number of virtual processors allocated to the virtual
+   machine under Settings --> System --> Processor for increased
    performance.
 
+#. Select :guilabel:`OK`.
```

Comment:
Makes sense to me. I don't know of any direct implications of this, but it might help avoid weird issues sticking to emulation of a modern chipset that we intentionally support vs. one from the mid-90s that we wouldn't support on bare metal.

Reply:
From the [manual](https://www.virtualbox.org/manual/ch03.html):

> You can select which chipset will be presented to the virtual machine. In legacy versions of Oracle VM VirtualBox, PIIX3 was the only available option. For modern guest OSes such as Mac OS X, that old chipset is no longer well supported. As a result, Oracle VM VirtualBox supports an emulation of the more modern ICH9 chipset, which supports PCI express, three PCI buses, PCI-to-PCI bridges and Message Signaled Interrupts (MSI). This enables modern OSes to address more PCI devices and no longer requires IRQ sharing. Using the ICH9 chipset it is also possible to configure up to 36 network cards, up to 8 network adapters with PIIX3. Note that the ICH9 support is experimental and not recommended for guest OSes which do not require it.

Reply:
@bwarden , thanks for looking into it

Reply:
@bwarden. Thanks for sharing that. Do you think a note/disclaimer is justifiable, given that ICH9 "support is experimental"? This could be as simple as: "Note: You can select which chipset will be presented to the virtual machine. Consult your manual for more details."

Reply:
Yeah, I think a note with a link to the manual is appropriate.